manifests/kustomize/base/metadata/metadata-deployment.yaml

```diff
@@ -18,17 +18,24 @@ spec:
       - name: container
         image: gcr.io/tfx-oss-public/ml_metadata_store_server:0.14.0
         env:
-        - name: GRPC_PORT
-          value: "8080"
-        - name: METADATA_STORE_SERVER_CONFIG_FILE
-          value: "/etc/config/mlmd_config.prototxt"
+        - name: DBCONFIG_USER
+          valueFrom:
+            secretKeyRef:
+              name: mysql-credential
+              key: username
+        - name: DBCONFIG_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: mysql-credential
+              key: password
+        command: ["/bin/metadata_store_server"]
+        args: ["--grpc_port=8080",
+               "--mysql_config_host=mysql",
```

Comment:
Is this true even if I'm using cloud sql?

Reply:
Yes, because even with Cloud SQL we have the service mysql running which points to the cloudsqlproxy deployment.

Reply:
Thanks for the clarification!